From 1984b1883dc964fd092160821e457dee394804bf Mon Sep 17 00:00:00 2001
From: Alin Dobre <alin.m.dobre@intel.com>
Date: Thu, 1 Mar 2012 15:11:48 +0200
Subject: Fix build failure when switching between build variants

When switching from the eng to the userdebug build variant, the init
binary compilation returns error ar linking stage:
.../obj/EXECUTABLES/init_intermediates/init.o: In function `bootchart_init_action':
system/core/init/init.c:729: undefined reference to `bootchart_init'
.../obj/EXECUTABLES/init_intermediates/init.o: In function `main':
system/core/init/init.c:881: undefined reference to `bootchart_step'
system/core/init/init.c:882: undefined reference to `bootchart_finish'
collect2: ld returned 1 exit status
make: *** [.../obj/EXECUTABLES/init_intermediates/LINKED/init] Error 1

Issue: ANDROID-1757
Change-Id: Ide50286e190ffeb75ec56bdfa2f5c612e0ae2392
Signed-off-by: Alin Dobre <alin.m.dobre@intel.com>
---
 CleanSpec.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/CleanSpec.mk b/CleanSpec.mk
index 8611d3b..f660d71 100644
--- a/CleanSpec.mk
+++ b/CleanSpec.mk
@@ -50,3 +50,4 @@
 
 $(call add-clean-step, rm -rf $(PRODUCT_OUT)/root/init.rc)
 $(call add-clean-step, rm -rf $(PRODUCT_OUT)/root/init.rc)
+$(call add-clean-step, rm -rf $(PRODUCT_OUT)/obj/EXECUTABLES/init_intermediates)
-- 
1.7.4.1

